src/commcare_cloud/ansible/roles/common/templates/etc_aliases.j2

```diff
@@ -19,6 +19,11 @@ couchdb: root
 rabbitmq: root
 kafka: root
 zookeeper: root
+monit: root
```

Comment:
The setting in `monitrc` correctly takes care of emails sent by monit. The alias in `/etc/aliases` makes sure that email _to_ monit also ends up in `root`'s mailbox.

There won't normally be email to `monit`, but I found at least one occasion grepping through my mail logs where the system sent it an email - and postfix failed to deliver.

Unfortunately this is a while ago and I didn't record what the email was about. It may have been a bounce or failure report for an outbound mail by monit.

The other two destinations that I found delivery failures for in the postfix logs are the ansible and cchq users, which is how I got to this trio.

Reply:
In fact, there is one more account: `ubuntu` aka the `{{ commcare_cloud_root_user }}` that is sometimes sent email, so you may want to also add:

```
{% if commcare_cloud_root_user is defined %}                                                                                                                                                                                                 
{{ commcare_cloud_root_user }}: root
{% endif %}
```